samples/msmq/sql-bridge/Core_5/SqlBridge/MsmqReceiver.cs

```diff
@@ -46,31 +48,50 @@ public Action<NServiceBus.Unicast.Transport.TransportReceiver> GetReceiverCustom
     // Will get invoked, whenever a new event is published by the Msmq publishers
     // and when they notify the bridge. The bridge is a MSMQ and the publishers
     // have an entry for this queue in their subscription storage.
-    public bool Handle(TransportMessage message)
+    public bool Handle(TransportMessage msmqMessage)
     {
-        var headers = message.Headers;
-        Type[] eventTypes =
+        var eventTypes = ExtractEventTypes(msmqMessage.Headers);
+        var sqlMessage = TranslateToSqlTransportMessage(msmqMessage);
+
+        log.Info("Forwarding message to all the SQL subscribers via a Publish");
+
+        publisher.Publish(sqlMessage, new PublishOptions(eventTypes.First()));
+        return true;
+    }
+
+    static Type[] ExtractEventTypes(Dictionary<string, string> headers)
+    {
+        return new []
         {
             Type.GetType(headers["NServiceBus.EnclosedMessageTypes"])
         };
+    }
 
+    static TransportMessage TranslateToSqlTransportMessage(TransportMessage msmqMessage)
+    {
+        var headers = msmqMessage.Headers;
         var msmqId = headers["NServiceBus.MessageId"];
+        var sqlId = msmqId;
```

Comment:
What's the purpose of the variable sqlId? The header assignment is only done if we have the need for a deterministic guid.